apps/contactsinteraction/lib/Listeners/UserDeletedListener.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+declare(strict_types=1);
+
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+namespace OCA\ContactsInteraction\Listeners;
+
+use OCA\ContactsInteraction\Db\RecentContactMapper;
+use OCP\EventDispatcher\Event;
+use OCP\EventDispatcher\IEventListener;
+use OCP\User\Events\UserDeletedEvent;
+
+/**
+ * @template-implements IEventListener<Event|UserDeletedEvent>
+ */
+class UserDeletedListener implements IEventListener {
+
+	public function __construct(
+		private readonly RecentContactMapper $recentContactMapper
```

Comment:
```suggestion
		private readonly RecentContactMapper $recentContactMapper,
```

for the cs fixer :grimacing:

Reply:
Done